lib/cdo/shared_constants.rb

```diff
@@ -265,6 +265,8 @@ module SharedConstants
       "buzzer.note": null,
       "buzzer.stop": null,
       "buzzer.play": null,
```

Comment:
This file is what actually defines the toolbox for the App Lab sandbox mode - remove `buzzer.play` here to remove it from the default App Lab toolbox.